source/mir/ndslice/slice.d

```diff
@@ -583,6 +491,195 @@ pure nothrow unittest
 }
 
 /++
+Creates an array and an n-dimensional slice over it.
+Params:
+    lengths = list of lengths for each dimension
+Returns:
+    n-dimensional slice
++/
+Slice!(Lengths.length, Select!(replaceArrayWithPointer, T*, T[]))
+createSlice(T,
+    Flag!`replaceArrayWithPointer` replaceArrayWithPointer = Yes.replaceArrayWithPointer,
```

Comment:
just out of interest? why no double quotes? `"`
Maybe we should start a style guide then ...